tests/test_backend.py

```diff
@@ -4105,6 +4105,54 @@ def func(x, y):
         self._run_test_case(func, [_OUTPUT], {_INPUT: input_x.astype(np.int32), _INPUT1: input_y}, as_session=True,
                             graph_validator=lambda g: check_op_count(g, "ConstantOfShape", 1, disabled=False))
 
+    @check_tf_max_version("2.13.1")
+    @check_opset_min_version(9, "ConstantOfShape")
+    @check_opset_max_version(9, "ConstantOfShape")
```

Comment:
Since we already have 2 functions for different versions, we don't need to add an opset limitation for these test methods. And we don't need to create different test cases for different versions. 

You just need to create one test case for tf op OnesLike, and the test framework will help on testing different ops.